runtimes/php/varnish.md

```diff
@@ -2,3 +2,107 @@
 title: Use Varnish as HTTP Cache
 shortdesc: This article shows you how enable Varnish on your application.
 ---
+
+# Varnish
+
+Varnish is a HTTP cache who works as a reverse proxy between the application server and the client, on the server-side.
```

Comment:
Varnish is a HTTP proxy-cache, which works as a reverse proxy between your application and the client.